ontologies/MIRO_sample_image_of_artwork.json

```diff
@@ -0,0 +1,93 @@
+; Comment: next step I assume would be to flesh out necessary context for this, including something to map these more human-readable terms to the ones in the .ttl ontologies
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "identifier": "ia2377yhy",
+  "type": "Work",
+  "sourceIdentifiers": [
+    {
+      "label": "V0006724",
+      "authority": "MIRO"
+    }
+    {
+      "label": "ICV No 6939",
+      "authority": "Videodisk"
+    }
+    {
+      "label": ".b11579328",
+      "authority": "Sierra"
+    }
+  ]
+  "licence": "CC-BY",
+  "label": "A lecture on pneumatics at the Royal Institution, London. Co",
+  "description": "A lecture on pneumatics at the Royal Institution, London. Coloured etching by J. Gillray, 1802.",
+  "lettering": "Scientific researches! -New discoveries in pneumaticks! -or- an experimental lecture on the powers of air.- Js. Gillray inv. & fect."
+  "creators": [
+      {
+      "label": "James Gillray",
+      "type": "Agent"
+      }
+      ]
+  "publishers": [
+      {
+      "label": "H. Humphrey,",
+      "type": "Agent"
+      }
+      ]
+  "placeofpublication": [
+      {
+      "label": "[London] (St. James's Street)",
+      "type": "Place"
+      }
+      ]
+  "createdDate": "1802",
+  "publicationdate": "May 23 1802",
+; Comment: MIRO data can include both a createdDate field generated by a MIRO cataloguer and a publicationDate field pulled across from Sierra (where appropriate): suggest where the latter exists it should take priority?
+  "format": "etching, with watercolour",
+  "physicaldescription": "Paper",
+  "genre": "Caricatures"
+  "subjects": [
+; Comment: I've not yet set up the Subjects ontology but am assuming that each subject will have a note of its label and its source authority.  In MIRO there are keywords that don't come from any external thesaurus - do we want to display these?
+      {
+      "label": "Caricatures",
+      "authority": "MeSH" 
```

Comment:
Medical Subject Headings - the default thesaurus within the Library's catalogues.  You can see it at https://meshb.nlm.nih.gov/search.  The other thesaurus you'll see us using, when MeSH doesn't have the required detail (broadly, for stuff that's less strictly medical and/or older) is LCSH, Library of Congress Subject Headings (http://authorities.loc.gov/cgi-bin/Pwebrecon.cgi?DB=local&PAGE=First).